### PR TITLE
ledge-qemuarm change to enable secure boot

### PR DESCRIPTION
**Pull request by muvarov**
stage one of patches. It hangs somewhere on transition to BL2 to BL3. After debugging will send correction patch.

### CODE REVIEW

File: meta-ledge-bsp/recipes-security/optee/optee-os_git.bbappend [hunk 1]
```diff
@@ -17,21 +17,21 @@ OPTEEOUTPUTMACHINE_ledge-ti-am572x = "ti"
 EXTRA_OEMAKE_remove_ledge-ti-am572x = "CFG_ARM64_core=y"
 EXTRA_OEMAKE_remove_ledge-ti-am572x = "ta-targets=ta_arm64"
 EXTRA_OEMAKE_append_ledge-ti-am572x = "CFG_ARM32_core=y ta-targets=ta_arm32 "
-
 EXTRA_OEMAKE_append_ledge-ti-am572x = " CROSS_COMPILE_ta_arm32=${HOST_PREFIX} CROSS_COMPILE=${CROSS_COMPILE} "
 
+# armv7
+EXTRA_OEMAKE_remove_armv7a = "CFG_ARM64_core=y"
```
Comment (muvarov) [inline meta-ledge-bsp/recipes-security/optee/optee-os_git.bbappend]: that looks very strange....

File: meta-ledge-bsp/recipes-bsp/u-boot/u-boot-ledge-qemu/ledge-qemuarm_defconfig [hunk 2]
```diff
@@ -232,7 +232,7 @@ CONFIG_BOOTSTAGE_STASH_SIZE=0x1000
 CONFIG_BOOTDELAY=0
 # CONFIG_USE_BOOTARGS is not set
 CONFIG_USE_BOOTCOMMAND=y
-CONFIG_BOOTCOMMAND="setenv kernel_addr_r 0x60000000; setenv bootargs 'rootwait root=PARTLABEL=rootfs'; efidebug boot add 0000 'kernel' virtio 0:1 /efi/boot/bootarm.efi; efidebug boot order 0000; bootefi bootmgr"
+CONFIG_BOOTCOMMAND="load virtio 0 0x70000000 kernel.auth; setenv -e -nv -bs -rt -at -i 0x70000000,$filesize db; load virtio 0 0x70000000 KEK.auth; setenv -e -nv -bs -rt -at -i 0x70000000,$filesize KEK; load virtio 0 0x70000000 PK.auth; setenv -e -nv -bs -rt -at -i 0x70000000,$filesize PK; setenv kernel_addr_r 0x60000000; setenv bootargs 'rootwait root=PARTLABEL=rootfs'; efidebug boot add 0000 'kernel' virtio 1:1 /efi/boot/bootaa64.efi; efidebug boot order 0000; bootefi bootmgr"
```
Comment (cpriouzeau) [inline meta-ledge-bsp/recipes-bsp/u-boot/u-boot-ledge-qemu/ledge-qemuarm_defconfig]: Why do you  execute a aarch64 kernel for a 32bit platform ?

Comment (muvarov) [reply meta-ledge-bsp/recipes-bsp/u-boot/u-boot-ledge-qemu/ledge-qemuarm_defconfig]: thanks, copy-paste. But I did not go to that stage yet :)

Comment (muvarov) [reply meta-ledge-bsp/recipes-bsp/u-boot/u-boot-ledge-qemu/ledge-qemuarm_defconfig]: fixed in v2

**Pull request merged**